src/foam/nanos/notification/NotificationCitationView.js

```diff
@@ -13,40 +13,119 @@ foam.CLASS({
     foam.pattern.Faceted.create()
   ],
 
-  properties: [
-    'of',
+  css: `
+    ^ {
+      line-height: 17px;
+    }
+    ^ .initiationDescription {
+      font-family: IBMPlexSans;
+      font-size: 14px;
+      font-weight: 600;
+      color: #1e1f21;
+      margin-left: 16px;
+    }
+    ^ .created {
```

Comment:
Better to do ^created instead of ^ .created because if there are any nested views that also use the 'created' class, then it will conflict.